.env.example

```
@@ -1,20 +1,51 @@
+
+# 🌐 CONFIGURAÇÃO DE PROXY/VPN (Para contornar geo-bloqueio)
+# Opção 1: SOCKS5 Proxy (Recomendado)
+PROXY_SOCKS5_HOST=your_socks5_host
+PROXY_SOCKS5_PORT=1080
+
+# Opção 2: HTTP Proxy (Alternativa)
+PROXY_URL=http://username:password@proxy-host:port
+
+# 📊 PARÂMETROS DE TRADING REAIS
+PAIRS=BTC/USDT,ETH/USDT,BNB/USDT,SOL/USDT,ADA/USDT
+BASIS_ENTRY=0.001
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The BASIS_ENTRY key should go before the PAIRS key

(UnorderedKey)